src/js/ripple/base.js

```diff
@@ -145,16 +133,10 @@ Base.decode_check = function(version, input, alphabet) {
   }
 
   // Multiple allowed versions
-  if (Array.isArray(version)) {
-    var match = false;
-
-    for (var i = 0, l = version.length; i < l; i++) {
-      match |= version[i] === buffer[0];
-    }
-
-    if (!match) {
-      return NaN;
-    }
+  if (Array.isArray(version) && _.every(version, function(v) {
+          return v !== buffer[0];
```

Comment:
Yes, ideally it would be written as
`if (Array.isArray(version) && _.every(version, v => v !== buffer[0]))`